lib/localization/profanity/profanity_es.dart

```diff
@@ -0,0 +1,80 @@
+// coverage:ignore-file
+import 'profanity.dart';
+
+/// English profanity list.
+class ProfanityEs extends Profanity {
+  /// Creates a new instance of the profanity list.
+  ProfanityEs([super.locale = 'es']);
+
+  @override
+  List<String> get profanityList => <String>[
+        'Asesinato',
+        'asno',
+        'bastardo',
+        'Bollera',
+        'Cabrón',
+        'Caca',
+        'Chupada',
+        'Chupapollas',
+        'Chupetón',
+        'concha',
+        'Concha de tu madre',
+        'Coño',
+        'Coprofagía',
+        'Culo',
+        'Drogas',
+        'Esperma',
+        'Fiesta de salchichas',
+        'Follador',
+        'Follar',
+        'Gilipichis',
+        'Gilipollas',
+        'Hacer una paja',
+        'Haciendo el amor',
+        'Heroína',
+        'Hija de puta',
+        'Hijaputa',
+        'Hijo de puta',
+        'Hijoputa',
+        'Idiota',
+        'Imbécil',
+        'infierno',
+        'Jilipollas',
+        'Kapullo',
+        'Lameculos',
+        'Maciza',
+        'Macizorra',
+        'maldito',
+        'Mamada',
+        'Marica',
+        'Maricón',
+        'Mariconazo',
+        'martillo',
+        'Mierda',
+        'Nazi',
+        'Orina',
+        'Pedo',
+        'Pendejo',
+        'Pervertido',
+        'Pezón',
+        'Pinche',
+        'Pis',
+        'Prostituta',
+        'Puta',
+        'Racista',
+        'Ramera',
+        'Sádico',
+        'Semen',
+        'Sexo',
+        'Sexo oral',
+        'Soplagaitas',
+        'Soplapollas',
+        'Tetas grandes',
+        'Tía buena',
+        'Travesti',
+        'Trio',
+        'Verga',
+        'vete a la mierda',
+        'Vulva'
```

Comment:
There are a lot of contexts in which these words have no profane sense. 

I see no point in having a default word list. 
Having these words as a base will not save time for the developer who wants to make a quality or meaningful filter.